ui/public/i18n/language.json

```
@@ -495,7 +495,8 @@
         "Synchronization_info":"Synchronization information",
         "synchronization_informations_ok": "Synchronization information has been updated",
         "synchronization_informations_error": "Synchronization information has not been updated",
-        "Exclude": "Exclude folders"
+        "Exclude": "Exclude folders",
+        "TrashSync": "Synchronise the Trash"
```

To keep the JSON patch clean, add the new entry before the last one. Also fix the US spelling for `synchroni(s|z)e`

```suggestion
        "TrashSync": "Synchronize the Trash",
        "Exclude": "Exclude folders"
```